src/PhpWord/Writer/Word2007/Part/Chart.php

```diff
@@ -406,11 +406,11 @@ private function writeAxisTitle(XMLWriter $xmlWriter, $title)
         $xmlWriter->startElement('c:title'); //start c:title
         $xmlWriter->startElement('c:tx'); //start c:tx
         $xmlWriter->startElement('c:rich'); // start c:rich
-        $xmlWriter->writeElementBlock('a:bodyPr');
-        $xmlWriter->writeElementBlock('a:lstStyle');
+        $xmlWriter->writeElementBlock('a:bodyPr', []);
+        $xmlWriter->writeElementBlock('a:lstStyle', []);
         $xmlWriter->startElement('a:p');
         $xmlWriter->startElement('a:pPr');
-        $xmlWriter->writeElementBlock('a:defRPr');
+        $xmlWriter->writeElementBlock('a:defRPr', []);
```

Comment:
We need to either:
1) Update minimum php version in composer json (Currently 5.3.3) to 5.6.0
2) Use legacy array syntax that is compatible with PHP 5.3.3

Reply:
Interesting catch.  I'll take the composer 5.6.0 tactic.